GUIDE.md

```diff
@@ -0,0 +1,81 @@
+- A PC running any OS capable of running Node.JS that is connected to a network your Switch can connect to
+
+Setting up the Environment
+--------------------------
+### Windows
+Windows users will need to setup the Windows Subsystem for Linux (WSL). Instructions on how to set that up can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). After WSL has been setup, continue with the Linux instructions, using the Ubuntu/Debian commands.
+### Linux
+You will need to install `git` and `nodejs` from your distributions package manager. Please note that the version of NodeJS installed must be 8.x.x or higher.
+- **Debian-based (Ubuntu, Mint, etc.)**: An alternate repository is required, as Ubuntu/Debian ships very old versions of NodeJS.
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install nodejs git build-essential
+```
+- **Fedora**:
+```bash
+sudo dnf install nodejs npm git
+```
+- **Arch Linux**:
+```bash
+sudo pacman -S nodejs npm git base-devel
+```
+- **Other**: See the [installation guide for Node.JS](https://nodejs.org/en/download/package-manager/) to install Node.JS. Consult your distributions package manager to find packages for `git` and `nodejs` if the installation guide doesn't support it.
+### OS X/macOS
+You will need to install [Homebrew](https://brew.sh/) to download the necessary packages. After Homebrew is installed:
+```bash
+brew install node git
+```
+Setting up Pegaswitch
+---------------------
+1. In a terminal window, navigate to the location you want to install Pegaswitch to, and clone the Pegaswitch git repository using `git clone https://github.com/reswitched/pegaswitch.git`.
+2. Navigate into the new `pegaswitch` directory, and run `npm i`. This will install all the modules needed for Pegaswitch to function.
+3. Start Pegaswitch with `sudo node start --setuid $(id)`. Your user id can be found by running `id`, and searching the number next to your username (typically 1000, on a single-user system).
+	- If you are using Puyo Puyo Tetris or fakenews to access Pegaswitch, add `--webapplet` to this command. 
+
+Launching Pegaswitch
+--------------------
+4. Setup a connection to the Internet on your Switch, if you haven't already. Don't stay connected for too long without setting the DNS, otherwise your Switch may download an update.
+5. On your Switch, go into Settings -> Internet -> Internet Settings, pick the network your computer running Pegaswitch is connected to, and select Change Settings.
+6. Select DNS Settings and change it from Automatic to Manual. Set the Primary and Secondary DNS settings to the IP address that Pegaswitch shows in it's console.
+7.  - 1.0.0: Follow the instructions [here](http://switchbrew.org/index.php?title=Internet_Browser#WebApplet_launch_with_Tetris) to launch Pegaswitch with Puyo Puyo Tetris. If you 	don't feel like running PPT everytime you want to use pegaswitch see fakenews below.
```

Comment:
"If you don't feel like running PPT everytime you want to use pegaswitch see fakenews below" -> "After launching PegaSwitch using Puyo Puyo Tetris, Fakenews can be used to install a faster way to launch PegaSwitch."